ew/cmd/fish/fishutils.py

```diff
@@ -511,6 +515,27 @@ async def award_fish(fisher, cmd, user_data):
         fisher.stop()
 
         user_data.persist()
+
+        #GoonScape Stat
+        xp_map = {
+            "item": 21000,
+            "common": 11000,
+            "uncommon": 16000,
+            "rare":	26000,
+            "promo": 31000,
+        }
+
+        if fisher.inhabitant_id:
+            xp_yield = xp_map.get(xp_type/2, 16000/2)
```

Comment:
you only need to assign this once, and also why are you dividing a string by 2 and using a second argument in .get? .get needs the rarity of the fish only